app/assets/stylesheets/school-bookings.scss

```diff
@@ -36,11 +36,11 @@
 
       .govuk-se-button-container {
         .govuk-button--secondary {
-          background-color: govuk-colour('grey-4');
+          background-color: govuk-colour('light-grey');
           color: govuk-colour('black');
 
           &:hover {
-            background-color: govuk-colour('grey-3');
+            background-color: darken(govuk-colour('light-grey'), 10%);
```

Comment:
I'm wary of us choosing our own colour schemes - would we not be better off picking one of the other existing 'grey' colours?

Do we even need to be specfiying colours? Shouldn't these becoming from frontend itself for `btn-secondary` ? Or is this it doesn't supply a secondary button?

Reply:
Yeah, you're right. Secondary buttons are now part of the design system, I'll remove this